docs/tutorials/aws.md

```diff
@@ -65,7 +65,8 @@ In this case it's the ones shown above but your's will differ.
 ## Deploy ExternalDNS
 
 Connect your `kubectl` client to the cluster you want to test ExternalDNS with.
-Then apply one of the following manifests file to deploy ExternalDNS.
+Then apply one of the following manifests file to deploy ExternalDNS. 
+You can check if your cluster has RBAC by `kubectl api-version`, `kubectl api-versions | grep rbac.authorization.k8s.io`.
```

Comment:
I would drop this `kubectl api-version` and only use `kubectl api-versions | grep rbac.authorization.k8s.io`.

Reply:
@njuettner I decided to open new PR because I kinda screw up with CLA
https://github.com/kubernetes-incubator/external-dns/pull/959